revolv/static/auth.css

```diff
@@ -99,3 +90,31 @@ small.field-error {
     -webkit-transition: color 0.1s; }
   .submit-button:disabled {
     opacity: 0.3; }
+  .submit-button.gray {
+    cursor: pointer;
+    background-color: transparent;
+    color: #b3b3c0;
+    border: 1px solid #b3b3c0; }
+    .submit-button.gray:hover {
+      background-color: #b3b3c0;
+      color: white;
+      transition: background-color 0.2s;
+      transition: color 0.1s;
+      -webkit-transition: background-color 0.2s;
+      -webkit-transition: color 0.1s; }
+    .submit-button.gray:disabled {
+      opacity: 0.3; }
+  .submit-button.yellow {
+    cursor: pointer;
+    background-color: transparent;
+    color: #f0ad4e;
+    border: 1px solid #f0ad4e; }
+    .submit-button.yellow:hover {
+      background-color: #f0ad4e;
+      color: white;
+      transition: background-color 0.2s;
+      transition: color 0.1s;
+      -webkit-transition: background-color 0.2s;
+      -webkit-transition: color 0.1s; }
```

Comment:
there's a lot of repeated styles in the .submit-button-gray and the .submit-button-yellow! maybe they could be consolidated?